backend/.eslintrc.js

```diff
@@ -1,8 +1,17 @@
 module.exports = {
   env: {
-    node: true
+    node: true,
+    jest: true,
   },
   rules: {
     "no-unused-expressions": ["off"]
-  }
+  },
+  overrides: [
+    {
+      files: ['*.test.js', '*.spec.js'],
+      rules: {
+        "import/no-extraneous-dependencies": "off"
```

Comment:
Porque precisou desabilitar essa regra? Ela não é importante para evitar importações erradas?

Reply:
Não, eu só trouxe pra cá por conta da configuração do eslint que estava nos testes. Ela é usada no  `office.socket.test.js`, na importação do ` require("socket.io-client")`. Talvez de pra remover isso e deixar desabilitado só onde for necessário. Assim não incentiva a galera a fazer isso.